app/main.py

```diff
@@ -1 +1,25 @@
-# write your code here
+def copy_file(command: str) -> None:
+    try:
+        cmd, old_file, new_file = command.split()
+    except ValueError:
+        print("You put incorrect data")
+    else:
+        try:
+            if old_file == new_file:
```

Comment:
You can also skip raising an error here and just use reversed condition according to the task condition